all_repos/bitbucket_server_api.py

```diff
@@ -0,0 +1,32 @@
+import json
+import urllib.request
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+
+class Response(NamedTuple):
+    values: Any
+    next: Optional[int]
+    links: Any
+
+
+def req(url: str, **kwargs: Any) -> Response:
+    resp = urllib.request.urlopen(urllib.request.Request(url, **kwargs))
+    obj = json.load(resp)
+    next_index = None
+    if obj.get('nextPageStart') is not None and obj.get('isLastPage') is False:
```

Comment:
if the first key is there, is the second key guaranteed? if so you can write `and not obj['isLastPage']`

Reply:
I think it should be as both are part of the paging meta data - though I did not want to assume so. I'll change it.